.github/workflows/release.yml

```
@@ -353,3 +356,105 @@
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set Tag Version
+        run: |
+          TAG_VERSION="${{ needs.release.outputs.tag }}"
+          echo "TAG_VERSION=$TAG_VERSION" >> $GITHUB_ENV
+          echo "RELEASE_BUILD=true" >> $GITHUB_ENV
+          echo "🏷️ Building with tag: $TAG_VERSION"
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)